msal/src/main/java/com/microsoft/identity/client/PublicClientApplicationConfiguration.java

```diff
@@ -479,11 +484,10 @@ private void nullConfigurationCheck(String configKey, String configValue) {
         }
     }
 
-    private boolean isBrokerRedirectUri() {
-        final String BROKER_REDIRECT_URI_REGEX = "msauth://" + mAppContext.getPackageName() + "/.*";
-        final Pattern pairRegex = Pattern.compile(BROKER_REDIRECT_URI_REGEX);
-        final Matcher matcher = pairRegex.matcher(mRedirectUri);
-        return matcher.matches();
+    @VisibleForTesting
+    public static boolean isBrokerRedirectUri(final @NonNull String redirectUri, final @Nullable String packageName) {
```

Comment:
Does the package name have to be nullable?

Reply:
There's no requirement for the restriction to NonNull.

Reply:
So what should be return from this method if package name is null? I don't see a null check here

Reply:
It doesn't matter, so I'm making it @NonNull